framework/packages/abstract-sdk/src/apis/ibc.rs

```diff
@@ -87,11 +87,11 @@ pub struct IbcClient<'a, T: IbcInterface> {
 impl<'a, T: IbcInterface> IbcClient<'a, T> {
     /// Get address of this module
     pub fn module_address(&self) -> AbstractSdkResult<Addr> {
```

Comment:
Could we rename this to `address` instead of `module_address`?

Reply:
We use `module_address` in `Modules` and `CronCat` API's as well! Would you prefer to use `address` there too?